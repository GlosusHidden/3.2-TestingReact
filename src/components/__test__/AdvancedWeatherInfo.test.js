import React from 'react'
import renderer from 'react-test-renderer'

import AdvancedWeatherInfo from '../AdvancedWeatherInfo'

import {myStore} from './test_data/data'

it('AdvancedWeatherInfo Component Snapshot', () => {
  const renderedValue = renderer.create(
    <AdvancedWeatherInfo data={myStore.action.Paris}/>
  ).toJSON();
  expect(renderedValue).toMatchSnapshot();
});
