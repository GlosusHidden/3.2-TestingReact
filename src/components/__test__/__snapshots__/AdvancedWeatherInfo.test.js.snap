// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AdvancedWeatherInfo Component Snapshot 1`] = `
<div
  className="list"
  id="advanced_weather_info"
>
  <div
    className="line"
  >
    <div
      className="param"
    >
      Weather
    </div>
    <div
      className="value"
    >
      Light rain
    </div>
  </div>
  <div
    className="line"
  >
    <div
      className="param"
    >
      Pressure
    </div>
    <div
      className="value"
    >
      753 mmHg
    </div>
  </div>
  <div
    className="line"
  >
    <div
      className="param"
    >
      Humidity
    </div>
    <div
      className="value"
    >
      93 %
    </div>
  </div>
  <div
    className="line"
  >
    <div
      className="param"
    >
      Wind
    </div>
    <div
      className="value"
    >
      2.1
    </div>
  </div>
  <div
    className="line"
  >
    <div
      className="param"
    >
      Coordinates
    </div>
    <div
      className="value"
    >
      [48.86, 2.35]
    </div>
  </div>
</div>
`;
